tests/e2e/test_migrating/migration_signed_invokes.cairo

```diff
@@ -5,7 +5,7 @@ func up() {
     %{
         contract  = deploy_contract("./build/main.json")
 
-        invoke(contract.contract_address, "put", { "value": 69 }, config={"auto_estimate_fee": True})
+        invoke(contract.contract_address, "put", { "value": 69 }, config={"max_fee": "auto"})
```

Comment:
I think Protostar should check if auto_estimate_fee exists. If so, tell the user to use `max_fee="auto"`.

(Btw. In the migration documentation, we have a warning `Breaking changes can be introduced without deprecation.`)

Reply:
[add warning about auto_estimate_fee deprecation](https://github.com/software-mansion/protostar/pull/979/commits/d41edcf97aae14c0562b2c82fbb576e5aa91c62d)